web/ui/react-app/src/pages/Config.tsx

```diff
@@ -1,6 +1,46 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Alert, Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import CopyToClipboard from 'react-copy-to-clipboard';
 
-const Config: FC<RouteComponentProps> = () => <div>Config page</div>
+import './Config.css';
+
+const Config: FC<RouteComponentProps> = () => {
+  const [config, setConfig] = useState(null);
+  const [error, setError] = useState("");
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    fetch('../api/v1/status/config')
+      .then(res => res.json())
+      .then(res => setConfig(res.data.yaml))
+      .catch(error => setError(error.message));
+  });
+
+  return (
+    <>
+      <h2>
+        Configuration&nbsp;
+        <CopyToClipboard
+          text={config ? config! : ''}
+          onCopy={() => { setCopied(true); setTimeout(() => setCopied(false), 1500);}}
+        >
+          <Button color="light" disabled={!config}>
+            {copied ? 'Copied' : 'Copy to clipboard'}
+          </Button>
+        </CopyToClipboard>
+      </h2>
+
+      {error
```

Comment:
I think Alert should be returned before everything else in case of error, wdyt?

Reply:
I had played around with this and preferred the error to appear in the place where the spinner is, and where the config would have appeared if there was no error. Then at least there's still the headline that tells you it's supposed to be the config page.